articles/reactive-distinctby.md

```diff
@@ -4,3 +4,5 @@ title: DistinctBy
 ---
 
 ![Marble diagram](~/images/reactive-distinctby.svg)
+
```

Comment:
For this one, it would be nice to include a little motif since it requires a somewhat "complex" input.

Reply:
I have opened a separate issue just for thinking about a good example for the *By operators: https://github.com/bonsai-rx/docs/issues/72